package.json

```diff
@@ -146,7 +146,7 @@
     "vsce": "^1.53.2"
   },
   "dependencies": {
-    "find-java-home": "0.2.0",
+    "locate-java-home": "^1.1.2",
```

Comment:
this is the main change. https://github.com/jvilk/locate-java-home can successfully detect all versions installed on the system, allowing us to cherry-pick java 8 if available.

On the other hand `find-java-home` returned a spurious value `~/.jenv` when using jenv and launching VSCode with `code /path/to/project`. A workaround was to launch VSCode with `jenv exect code /path/to/project`, but it wasn't really acceptable for end users.

Reply:
cc/ @laughedelic might be relevant for Atom